neptune/alpha/internal/backends/neptune_backend_mock.py

```diff
@@ -56,9 +56,32 @@ def get_project(self, project_id: str) -> Project:
         return Project(uuid.uuid4(), "sandbox", "workspace")
 
     def create_experiment(self, project_uuid: uuid.UUID) -> Experiment:
-        new_uuid = uuid.uuid4()
-        self._experiments[new_uuid] = ExperimentStructure[Value]()
-        return Experiment(new_uuid, "SAN-{}".format(len(self._experiments) + 1), project_uuid)
+        new_experiment_uuid = uuid.uuid4()
+        self._experiments[new_experiment_uuid] = ExperimentStructure[Value]()
+        return Experiment(new_experiment_uuid, "SAN-{}".format(len(self._experiments) + 1), project_uuid)
+
+    def get_experiment_with_attributes(self, project_id: str, experiment_id: str) -> (Experiment, List[Attribute]):
+        project_uuid = uuid.uuid4()
+        new_experiment_uuid = uuid.uuid4()
+        self._experiments[new_experiment_uuid] = ExperimentStructure[Value]()
+        experiment = Experiment(new_experiment_uuid, "SAN-{}".format(len(self._experiments) + 1), project_uuid)
+
+        attributes: List[Attribute] = []
+        attributes.append(Attribute("sys/id", AttributeType.STRING))
+        attributes.append(Attribute("sys/owner", AttributeType.STRING))
+        attributes.append(Attribute("sys/name", AttributeType.STRING))
+        attributes.append(Attribute("sys/description", AttributeType.STRING))
+        attributes.append(Attribute("sys/hostname", AttributeType.STRING))
+        attributes.append(Attribute("sys/creation_time", AttributeType.DATETIME))
+        attributes.append(Attribute("sys/modification_time", AttributeType.DATETIME))
+        attributes.append(Attribute("sys/size", AttributeType.FLOAT))
+        attributes.append(Attribute("sys/tags", AttributeType.STRING_SET))
+        attributes.append(Attribute("sys/notebook/id", AttributeType.STRING))
+        attributes.append(Attribute("sys/notebook/name", AttributeType.STRING))
+        attributes.append(Attribute("sys/notebook/checkpoint/id", AttributeType.STRING))
+        attributes.append(Attribute("sys/notebook/checkpoint/name", AttributeType.STRING))
```

Comment:
Why did you hardcoded some attributes instead of returning actual atributes from a strubture (like in get_attributes).